### DISCUSSION TITLE
What dose cloudflare know about my browser???

### DISCUSSION THREAD

**Discussion by NotYmL** [ultrafunkamsterdam/undetected-chromedriver#1863] [Q&A]
Some time ago, I saw this website https://www.amiunique.org/fingerprint, and I was wondering what information Cloudflare uses to fingerprint a browser and determine if it's a bot besides the headers that are sent in the initial request. 

Since Cloudflare runs some JS on my browser when I visit a protected website, I want to know what information Cloudflare looks for and sends to their servers.

If anyone can point me in the right direction to find more information on this topic, I would appreciate it.

**Comment by ChristianKleineidam** [comment]
I get the impression that since Chrome is showing "Chrome is being controlled by automated test software" when controlled via Selenium, Cloudflare is somehow also able to detect that it's controlled. That seems to be true, even if I manually click on the "I'm human" button from Cloudflare.
